README.md

```diff
@@ -48,6 +53,7 @@ Consult the [Roadmap](/ROADMAP.md) for a complete state description of the proje
       - [Files API](#files-api)
       - [Swarm API](#swarm-api)
       - [libp2p API](#libp2p-api)
+      - [Need the Buffer class?](#need-the-buffer-class)
```

Comment:
Maybe replace this by `"Domain data types"`, so that it includes things like multiaddr, multihash, etc.